lib/Mojo/WebSocketProxy/Backend/JobAsync.pm

```diff
@@ -37,11 +38,11 @@ Returns a new instance. Required params:
 
 =over 4
 
-=item loop => IO::Async::Loop
+=item loop => IO::Async::Loop (deprecate)
```

Comment:
why deprecated? shouldn't take any extra work to support these as parameters...

Reply:
As I remember  @nael-binary removed these args because they harmful effect on queue client shutdown. I'll try to test service shutdown by restoring these args and push changes if there is not any issue. @nael-binary Would you please give details about the reason behind these args and the test procedure if want to restore them?